chaining/challenges-10.test.js

```diff
@@ -0,0 +1,190 @@
+
+'use strict';
+
+// ------------------------------------------------------------------------------------------------
+// CHALLENGE 1
+//
+// Write a function named count that, given an integer and an array of arrays, uses either
+// filter, map, or reduce to count the amount of times the integer is present in the array of arrays.
+//
+// Note: You might need to use the same method more than once.
+//
+// For example, count(5, [[1, 3, 5, 7, 9], [5, 5, 5], [1, 2, 3]]) returns 4.
+// ------------------------------------------------------------------------------------------------
+
+const count = (target, input) => {
+  // Solution code here...
+  let amountOfTimesIntegerPresent = [];
+  input.map(num => {
```

Comment:
Not sure why you're using `map` here: you aren't returning anything from the callback and aren't trying to generate an array of the same length, so you probably just want to call `forEach`.